### PR TITLE
Fix code formatting in style guide

### PR DESCRIPTION
**Pull request by rhetoric101**
This PR is to fix our out-of-date examples for formatting code.

Here's what I did:

* I removed references to `<pre>` and `<code>` tags because those were a carryover from Drupal.
* I inserted steps for adding code formatting.

### CODE REVIEW

File: src/content/docs/style-guide/formatting/code-examples.mdx [hunk 1]
```diff
@@ -183,20 +208,11 @@ These sections break down when to use specific types of code formatting.
...
-
-    View a [full list of supported languages for syntax highlighting](/docs/new-relic-only/advanced-style-guide/design-specifications/supported-languages-syntax-highlighting). Element colors and themes are fully customizable.
-  </Collapser>
-
   <Collapser
     id="link-in-code-blocks"
     title="Link to other docs"
   >
-    You can add `<a href>` tags inside a `<pre>` blocks, just as you would any other content. However, make sure to use them judiciously since they can be easy to miss. A good example is the [Infrastructure config file template](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-config-file-template-newrelic-infrayml), where each config element links to the relevant section of the main config doc.
+    You can add `<a href>` tags inside a of your code blocks, just as you would any other content. However, make sure to use them judiciously since they can be easy to miss. A good example is the [Infrastructure config file template](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-config-file-template-newrelic-infrayml), where each config element links to the relevant section of the main config doc.
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/formatting/code-examples.mdx:215]: Remove the "a" after inside, or reword the sentence.

File: src/content/docs/style-guide/formatting/code-examples.mdx [hunk 2]
```diff
@@ -89,22 +89,47 @@ These sections break down when to use specific types of code formatting.
...
+
+  ![Screenshot showing how to format a handlebars code block](./images/code-examples.png "Screenshot showing how to format a handlebars code block")
+
+  This is what the formatting looks like when it's published:
+
+```handlebars
+{{json data.tags}}
+```
+
+  To figure out if the language you want to format is already set up in Gatsby, see if it's listed in `gatsby-config.js` under `prism`. The list of currently supported languages would look like this:
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/formatting/code-examples.mdx:113]: I'd be more straight forward.

Check in `gatsby-config.js` (under `prism`) if the language you want to format is already set up in Gatsby. The list of currently supported languages is:

**Review by urbiz-nr — DISMISSED** [review]
Nice! Left a couple of comments, up to you!

File: src/content/docs/style-guide/formatting/code-examples.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/formatting/code-examples.mdx]: @urbiz-nr, good catch! I'll fix this!

File: src/content/docs/style-guide/formatting/code-examples.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/formatting/code-examples.mdx:113]: Thanks, @urbiz-nr !

**Comment by rhetoric101** [comment]
I am closing this because Austin was doing parallel work on this topic.

**Pull request closed without merging**